File: articles/algorithms-in-labor-management-and-organizational-systems/prolog/workload_burden.pl
workload_burden(Pace, Hours, Fatigue, ScheduleVolatility, Burden) :-
    Burden is (Pace + Hours + Fatigue + ScheduleVolatility) / 4.

% Query example:
% ?- workload_burden(0.84, 0.72, 0.70, 0.78, Burden).

File: articles/algorithms-in-labor-management-and-organizational-systems/haskell/WorkloadBurden.hs
module Main where

main :: IO ()
main = do
  let pace = 0.84
      hours = 0.72
      fatigue = 0.70
      scheduleVolatility = 0.78
      burden = (pace + hours + fatigue + scheduleVolatility) / 4.0
  putStrLn ("workload_burden_score=" ++ show burden)
